app/src/main/res/values/strings.xml

```diff
@@ -210,6 +210,8 @@
     <string name="asset_download_dialog_open_text">Open</string>
     <string name="asset_download_dialog_save_text">Save</string>
     <string name="member_name_deleted_label">Deleted User</string>
+    <string name="member_name_you_label_titlecase">You</string>
+    <string name="member_name_you_label_lowercase">you</string>
```

Comment:
Do we need to store the lowercase? What do you think of simply making it lowercase when you need it?

Reply:
We need to store it because for messages `UIText` is used so we need to provide the StringRes Int and we can't modify the actual String, we would need to create another parameter in UIText to determine if it should be title or lowercase but it just doesn't make sense to add it for every UIText just because of this one case 🙈